doc-content/enterprise-only/openshift/template-deploy-rhsso-proc.adoc

```diff
@@ -69,24 +63,14 @@ Do not configure LDAP authentication and RH-SSO authentication in the same deplo
 .Prerequisites
 
 * A realm for {PRODUCT} is created in the RH-SSO authentication system.
-* User names and passwords for {PRODUCT} are created in the RH-SSO authentication system. For a list of the available roles, see <<roles-users-con>>.
-ifdef::sso_serveruser[]
-The following users are required in order to set the parameters for the environment:
-** An administrative user with the `kie-server,rest-all,admin` roles. This user can administer and use the environment. {KIE_SERVERS} use this user to authenticate with {LOCAL_CENTRAL}.
-** A server user with the `kie-server,rest-all,user` roles. This user can make REST API calls to the {KIE_SERVER}. {LOCAL_CENTRAL} uses this user to authenticate with {KIE_SERVERS}.
-endif::sso_serveruser[]
-ifndef::sso_serveruser[]
-In order to set the parameters for the environment, an administrative user with the `kie-server,rest-all,admin` roles is required. The default user name for this user is `adminUser`. This user can administer and use the environment.
-endif::sso_serveruser[]
+* User names and passwords for {PRODUCT} are created in the RH-SSO authentication system. For a list of the available roles, see <<roles-users-con>>. 
++
+As a minimum, you must create a user with the username and password configured in the secret for the administrative user, as described in <<secret-adminuser-create-proc>>. This user must have the `kie-server,rest-all,admin` roles.
```

Comment:
Minor rewrite suggestion:

You must create a user with the username and password configured in the secret **<MH> "secret" what? Is this a property?</MH>** for the administrative user, as described in <<secret-adminuser-create-proc>>. This user must have the `kie-server,rest-all,admin` roles.